web/ui/module/codemirror-promql/src/parser/parser.ts

```diff
@@ -275,6 +275,14 @@ export class Parser {
       }
     }
 
+    if (funcSignature.name === 'info') {
+      if (args.length > 1) {
```

Comment:
```suggestion
if (args.length > 1 && args[1].getChild(Identifier)) {
```

It will be more compact

Reply:
and maybe if you could add a comments to explain, it's to check that no metrics_name is used, that would to understand

Reply:
I think you've forgot to remove line 280?

Reply:
I guess I can commit your suggestion, and fix up.

Reply:
Done.